frontend/lib/src/components/shared/Dropdown/VirtualDropdown.tsx

```diff
@@ -61,16 +61,27 @@ const VirtualDropdown = React.forwardRef<any, any>((props, ref) => {
   if (!children[0] || !children[0].props.item) {
     const childrenProps = children[0] ? children[0].props : {}
     return (
-      <StyledList $style={{ height: `${EMPTY_LIST_HEIGHT}px` }} ref={ref}>
-        <StyledEmptyState {...childrenProps} />
+      <StyledList
+        $style={{ height: `${EMPTY_LIST_HEIGHT}px` }}
+        ref={ref}
+        data-testid="stVirtualDropdown"
```

Comment:
This feels like an exception since function conditionally returns - prefer calling this one 
`data-testid="stVirtualDropdownEmpty"` and keeping the normal one below `data-testid="stVirtualDropdown"`

Reply:
Should the data-testid below become `stVirtualDropdownEmptyStyledEmptyState`?

Reply:
I think the one directly below can be removed since we are making the distinction in StyledList 👍🏼